crates/sui-framework/packages/sui-framework/sources/crypto/zklogin_verified_id.move

```diff
@@ -69,9 +70,18 @@ module sui::zklogin_verified_id {
         audience: String,
         pin_hash: u256,
         ctx: &mut TxContext,
-    ): VerifiedID {
+    ) {
```

Comment:
Nit: might be shorter to put `tx_context::sender(ctx)` in a var, since we call it 3 times.